tests/test_other.py

```diff
@@ -9750,6 +9750,7 @@ def test_syslog(self):
     self.do_other_test('test_syslog.c')
 
   def test_split_module(self):
+    self.clear()
```

Comment:
This is not necessary since tests all start with a clean directory.

I think we just need to call `os.remove()` before `os.rename()`.  strange that windows python differs in this way..

Reply:
oh I think I just misunderstood what the test was doing...